readthedocs/core/templatetags/core_tags.py

```diff
@@ -112,3 +114,9 @@ def key(d, key_name):
 @register.simple_tag
 def readthedocs_version():
     return __version__
+
+
+@register.simple_tag
+def get_past_time(mins):
```

Comment:
What about naming this as `minutes_ago`?

Reply:
I was confused about what to call it. Thanks for the suggestion. I will change it as soon as possible.